Misc/NEWS.d/next/Tests/2019-04-28-18-36-54.bpo-36746.aqYNtS.rst

```diff
@@ -0,0 +1 @@
+A test for `fcntl.lockf()` is now implemented.
```

Comment:
NEWS entry isn't need for trivial PRs. You can submit your work without adding a NEWS entry. I'm working on a PR that sets the "skip news" label automatically for documentation and test only PRs.

Reply:
ohh great...At this point I dont know how to draw a line between trivial and non trivial but my judgement will get better with time.

Reply:
I usually ask myself questions like "would this change be important to users? can it break user code?" when I'm not sure about adding a NEWS entry. New features and bug fixes are always need a NEWS entry.

In PR #12999's case, while increasing test coverage of fcntl module is nice, it doesn't have any effect on end users, so we can merge it without a NEWS entry.

Reply:
Also, don't worry about NEWS entries too much, we can always add or delete them later :)

Reply:
Cool. Also I wanted to change the test for posix.lockf() to look as detailed as the one in this PR. Does this implementation look better ?